vllm/executor/ray_utils.py

```diff
@@ -101,24 +102,27 @@ def initialize_ray_cluster(
         # Verify that we can use the placement group.
         gpu_bundles = 0
         for bundle in bundles:
-            bundle_gpus = bundle.get("GPU", 0)
+            bundle_gpus = bundle.get(device_str, 0)
             if bundle_gpus > 1:
                 raise ValueError(
-                    "Placement group bundle cannot have more than 1 GPU.")
+                    "Placement group bundle cannot have more than 1 "
+                    f"{device_str}.")
             if bundle_gpus:
                 gpu_bundles += 1
         if parallel_config.world_size > gpu_bundles:
             raise ValueError(
-                "The number of required GPUs exceeds the total number of "
-                "available GPUs in the placement group.")
+                f"The number of required {device_str}s exceeds the total "
+                f"number of available {device_str}s in the placement group.")
```

Comment:
nit: but the error message will be nicer if we also include `parallel_config.world_size` and `gpu_bundles`. 

```
The number of required {device_str}s exceeds the total number of available {device_str}s in the placement groups. Required number of devices: {parallel_config.world_size}. Total number of devices: {gpu_bundles}.
```

Reply:
Sounds good! Added.